src/index.ts

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env node
 
 import * as program from "commander";
+import * as dotenv from "dotenv";
 import * as fs from "fs";
 import { slackRunner } from "./slack/slack-alert";
+
+dotenv.config();
```

Comment:
Hey,

Can you wrap this in an if statement and it wont interfere with the test

```
if (!process.env.CI) {
  config()
}
```

note in my example I used the import `import { config } from "dotenv";`

Thanks for your submission @ayushipatel126 , if you can make the changes and merge in master I will gladly get these changes in for you 👍🏾

Reply:
changes added to this PR https://github.com/YOU54F/cypress-slack-reporter/pull/988/commits

thank you for your submission @ayushipatel126